### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by basmilius** [basmilius/weather-icons#34]
<img width="973" alt="image" src="https://github.com/basmilius/weather-icons/assets/978257/e55823fa-8f4b-4573-a5e9-8c668454c264">

Hello everyone, it's been a while, but I'm excited to provide you with an update on the project's status. I want to express my gratitude for your support through stars and issue creation; your contributions mean a lot to me!

**What takes so long**
Regrettably, I've been facing time constraints when it comes to working on the project. My current commitments include a demanding full-time position at a company and running my own software development company for various clients, which consumes approximately 70 hours per week. I'm also striving to find some personal downtime. Nevertheless, I'm enthusiastic about dedicating more time to this project in the future, as I have ambitious plans for enhancements and additions.

**Migration to Figma**
I stopped using Adobe products because I moved over to Figma and keeping my Adobe license just for this projects is not a great idea. I'm currently in the process of re-creating all icons and adding some new ones in the process. The icons are animated using a great plugin called [Aninix](https://www.aninix.com/) in both animated svg's and lottie files. The WIP Figma file can be found [here](https://www.figma.com/file/RJabLIPrNCcnGYXP0wo3J9/Weather-Icons?type=design&node-id=0-1&mode=design).

**Updated animations**
Some icons will feature updated animations due to certain limitations that I didn't encounter in After Effects and SVG code. For instance, in the current version, raindrops continuously fall, but in the new version, they will intermittently pause.

- [New overcast day rain icon](https://app.aninix.com/inspect/lngd2cikglmnr1pbjbd/details)
- [Old overcast day rain icon](https://bmcdn.nl/assets/weather-icons/v3.0/fill/svg/overcast-day-rain.svg)

**3.0**
Lastly, I hope to provide you all with an update on a 3.0 release soon, which will include the updated icons and the new repository format. There is still much work to be done, including the line and monochrome styles. Thank you for your patience.

**Comment by basmilius** [comment]
There is an updated Figma file available on the following url: https://www.figma.com/design/y83khcCLLy5d8CYPMvo9p4/Weather-Icons-FigMotion

**Comment by simon041988** [comment]
First of all thank you very much for your great work! Could you give us an update on the status of V3.0? Thank you very much!

**Comment by codybrom** [comment]
Related, any way we can help?

**Comment by basmilius** [comment]
@simon041988 Yes, I'm currently animating all the icons using [Aninix](https://www.aninix.com/). It is taking some time, but I'm making good progress :)

@codybrom Not sure how currently, but thanks :)

**Comment by euskadi31** [comment]
Hello,

These icons are really great!

Do you cover all WMO Weather interpretation codes?

Can you give us an update on the progress?

Thank you

See: https://open-meteo.com/en/docs#weather_variable_documentation
See: https://gist.github.com/stellasphere/9490c195ed2b53c707087c8c2db4ec0c

**Comment by ygd** [comment]
@basmilius ping, if I can support you somehow to free a bit of your time. I am confident in design and code

**Comment by basmilius** [comment]
@ygd Helping is always appreciated of course! But the only thing that remains to do is animating them and that can't be done until [Aninix](https://www.aninix.com/) releases their new version.

You may of course look at the Figma design and share improvements or suggest changes :)

**Comment by ygd** [comment]
@basmilius I requested beta access to the new standalone app. let's see

**Comment by basmilius** [comment]
Most icons are now animated and I'm working on a documentation website instead of using my own personal website. The remaining icons that need to be animated are some specific ones. All animated icons are in both SVG and Lottie :)

I released the new documentation on https://meteocons.com already :)

**Comment by thasmo** [comment]
You're knocking it out of the park!

**Comment by bossanova808** [comment]
Can not believe the comment in that drizzle issue - please don't let it discourage you.  Your work here is awesome and much appreciated!

I personally will be revisiting my use of these icons in a little project soon (https://github.com/bossanova808/10-foot.dash) - but am currently holding off till you actually release the final V3.  But for some years now they've been giving me and a few folks pleasure on our little dashboards and, again, it really is much appreciated.

**Comment by brendanmatkin** [comment]
This is exciting and I love this library. Thank you and looking forward to the new release! Ping me if you need some testing?

**Comment by JoshuaKimsey** [comment]
I can't believe how far and how quickly (thanks to Claude) this has progressed! Following in the footsteps of bossanova808 , I want to show off where I'm using these icons (now the V3 version even!) as well, my little fun project called Variable Weather (https://github.com/JoshuaKimsey/variable-weather). The Meteocons were the absolute perfect fit for it, and with the new additions there's still more things I can switch to using the V3 icons with. A massive thanks to basmilius  for all the amazing work you've done on this, and for keeping it going as well! 🎉

**Comment by bossanova808** [comment]
@basmilius 

I have started to update my project to use the V3 icons, which has been quite simple so far thanks to your excellent documentation.  Thank you again!

I understand I could control animations if I use Lottie and will explore that if need be - however I think the _default_ animation speed is perhaps a bit too busy/fast now, with the SVGs - it was certainly a lot slower in previous versions.  E.g. a clear day - the 'sun lines' spin really quite quickly around the sun now, and just in general looking at https://v3.meteocons.com/ - I personally feel (but of course do recognise it's completely subjective!) - that the default speeds are a bit too much and attention grabbing for the sort of typical use case of weather icons (some sort of info display scenario).  

Just one person's opinion of course, but perhaps worth thinking about?
